Importer/CsvImporter.php

```diff
@@ -248,16 +241,18 @@ protected function checkHeadersAreValid($csv)
     }
 
     /**
-     * @param ImporterErrorInterface $error
+     * @param DBALException $exception
+     *
+     * @return string
      */
-    protected function handleImporterError(ImporterErrorInterface $error)
+    private function resolveDBALException(DBALException $exception)
```

Comment:
Seems logical for an importer to handle this. I'm just not sure if a CSV importer should? If we create a new import, then we have to define it all again.